### ISSUE TITLE
Associate DNS name with EC2 instance through config for bootstrap service

### ISSUE THREAD

**Issue by vuldin** [vuldin/redpanda-tf-asg#1]
Right now a Route53 record is updated with an Elastic IP address. Once the bootstrap EC2 instance is up and everything is deployed, the user has to take a manual step: open AWS Console and associate the Elastic IP with the EC2 instance. The issue is the EC2 instance is controlled by the autoscaling group.

Instead, this likely requires a load balancer and/or target group. More investigation is needed.

**Comment by vuldin** [comment]
Current approach is to directly update the DNS record with the bootstrap service's external IP within the user data script (focused on getting the IAM policy updates in place).

**Comment by vuldin** [comment]
I tested the time for the DNS name to become resolvable to the external IP address of the instance, and this took between 30 - 60 seconds which was too long. Instances would have to wait for the bootstrap service to be accessible before starting. I switched to the DNS entry being resolved to an elastic IP, and then associating this IP with the bootstrap instance from within the user data script. This results in the bootstrap service being resolvable in less than ~20 seconds.
